backend/tournesol/views/rate_later.py

```diff
@@ -64,6 +65,36 @@ def perform_create(self, serializer):
         self.prefetch_entity(rate_later)
 
 
+@extend_schema_view(
+    post=extend_schema(
+        description="Add a multiple new entities to the logged user's rate-later list,"
+        " for a given poll.",
+        request=RateLaterSerializer(many=True),
+        responses={
+            201: RateLaterSerializer,
```

Comment:
```suggestion
            201: RateLaterSerializer(many=True),
```

---

The API returns a list of `RateLater` serialized instances, but with this schema definition the front end will expect a single  `RateLater` element. This is not a problem for the web browser extension, but the generated service for the front end will be wrong.

Here is the currently generated service in the front end (file `src/services/openapi/services/UserService.ts`):

```typescript
    public static usersMeRateLaterBulkCreateCreate({
        pollName,
        requestBody,
    }: {
        pollName: string,
        requestBody: Array<RateLaterRequest>,
    }): CancelablePromise<RateLater> {
        return __request(OpenAPI, {
            method: 'POST',
            url: '/users/me/rate_later/{poll_name}/_bulk_create',
            path: {
                'poll_name': pollName,
            },
            body: requestBody,
            mediaType: 'application/json',
        });
    }
``` 

We expect the return type to be `CancelablePromise<Array<RateLater>>` or eventually `CancelablePromise<PaginatedRateLaterList>`.

Reply:
Good catch. Done in d4f78c4ce7fabd94e0ba97c651ac65cc9b40b6a6.